devel/gomp/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                gomp
+version             1.1.0
+platforms           darwin
+license             MIT
+categories          devel python
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         Git cOMPare
+long_description    {*}${description} - GOMP is a Git-based tool for \
+                    comparing branches, designed to work with a \
+                    rebase-oriented Git workflow. Markforged uses GOMP as \
+                    both a daily tool for handling commits and merges and as \
+                    a powerful way to visualize complex histories while \
+                    preparing releases.
+
+homepage            https://github.com/MarkForged/GOMP
+
+checksums           rmd160  e2eef3683d3a7bd544dcb175491dc94e69d6bb63 \
+                    sha256  0bc2e81e29b89613d7cab7d28df6eca9191b7af73facdf9e8fce2ffc2220d886 \
+                    size    6358
+
+python.default_version  38
+python.versions         38
+
+if {${name} ne ${subport}} {
```

Comment:
Ah, but there aren't any subports in this port. So ${name} does equal ${subport}. So this block is not entered. You want to remove this condition so that the dependency gets added.

Reply:
Got it; fixed.